### DISCUSSION TITLE
Copy my IP address to clipboard

### DISCUSSION THREAD

**Discussion by doefler** [johnlindquist/kit#994] [Share]
[Open my-ip in Script Kit](https://scriptkit.com/api/new?name=my-ip&url=https://gist.githubusercontent.com/doefler/c71b36afb7e9f8702cc86108244bf424/raw/236ee8e2de9bd01219dc696b43394dc1e34240e9/my-ip.js")

```js
// Name: My IP
// Description: Get my IP address and copy it to the clipboard
// Author: Kim Døfler

import "@johnlindquist/kit"

let ip = (await $`dig +short @ns1.google.com -t txt o-o.myaddr.l.google.com`)?._combined;
ip = ip.replaceAll(/"/gi, '').trim();
await copy(ip);
```

**Comment by phifa** [comment]
@doefler is this working for you?

**Reply by doefler** [reply]
Hey @phifa 

Yes, it works great on my Macbook (have not tested on Windows).

**Reply by daivy** [reply]
Works for me on Windows.
